AlgebraInLean/World0LeanIntroSOLUTIONS/Sheet3Solutions.lean

```diff
@@ -0,0 +1,143 @@
+/-
+This is a solutions sheet.
+-/
+
+import Mathlib.Tactic
+
+/-Credit for some exercises goes to Kevin Buzzard and his Formalizing Mathematics course OR to the
```

Comment:
This paragraph doesn't need to be used in every sheet. Once is sufficient.

Reply:
Paragraph deleted from Sheets 3-5